packages/core/src/fhir-deduplication/resources/medication.ts

```diff
@@ -28,35 +26,50 @@ export function groupSameMedications(medications: Medication[]): {
   rxnormMap: Map<string, Medication>;
   ndcMap: Map<string, Medication>;
   snomedMap: Map<string, Medication>;
-  remainingMedications: Medication[];
   refReplacementMap: Map<string, string[]>;
 } {
   const rxnormMap = new Map<string, Medication>();
   const ndcMap = new Map<string, Medication>();
   const snomedMap = new Map<string, Medication>();
   const refReplacementMap = new Map<string, string[]>();
-  const remainingMedications: Medication[] = [];
+
+  function removeOtherCodes(master: Medication): Medication {
+    const code = master.code;
+    const filtered = code?.coding?.filter(
+      coding =>
+        coding.system?.includes(SNOMED_CODE) ||
+        coding.system?.includes(SNOMED_OID) ||
+        coding.system?.includes(NDC_CODE) ||
+        coding.system?.includes(NDC_OID) ||
+        coding.system?.includes(RXNORM_CODE) ||
+        coding.system?.includes(RXNORM_OID)
```

Comment:
just to confirm, in your meds analysis, did you see any meds with the code system CVX? It should only be for immunizations but just want to make sure

Reply:
i didn't include cvx as a possibility when i was doing analysis for meds, so i have no answer to that rn